### PR TITLE
Welcome first-time contributors (Lombiq Technologies: OCORE-154)

### PR DESCRIPTION
**Pull request by Piedone**
Part of https://github.com/OrchardCMS/OrchardCore/issues/15029. Add automatic welcome comments for people's first issue and first PR. When their first PR is merged, add another congrats comment, which also adds them as a contributor with the All Contributors Bot.

- This is how it looks under an issue: https://github.com/Lombiq/OCORE-154-workflows-test/issues/3.
- This is how it looks under a PR: https://github.com/Lombiq/OCORE-154-workflows-test/pull/1.

**Note to reviewers:** https://github.com/OrchardCMS/OrchardCore/pull/15706 needs to be merged first, since this links to a new docs page created there.

### CODE REVIEW

File: .github/workflows/first_time_contributor.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    # We don't use the actions/first-interaction action because it can't reference the author, nor can it comment after
+    # PR merge.
+    # Once https://github.com/wow-actions/welcome/pull/12 is merged, we should use https://github.com/wow-actions/welcome
+    # directly.
+    - uses: OrchardCMS/welcome-action@task/update-to-node-20
+      with:
+        FIRST_ISSUE_COMMENT: >
+          Thank you for submitting your first issue, awesome! 🚀 If you haven't filled out the template fully already,
+          please be sure to do so. This helps us better understand your feature request/bug report. A core team member 
+          will be with you shortly to review your issue.
```
Comment (MikeAlhayek) [inline .github/workflows/first_time_contributor.yml:22]:
```suggestion
          Thank you for submitting your first issue! 🚀 We're thrilled to receive your input. If you haven't completed the template yet, please take a moment to do so. This ensures that we fully understand your feature request or bug report. A core team member will review your issue and get back to you.
```

File: .github/workflows/first_time_contributor.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        FIRST_PR_COMMENT: >
+          Thank you for submitting your first pull request, awesome! 🚀 If you haven't read it already, please be sure 
+          to check out our [contribution guide](https://docs.orchardcore.net/en/latest/docs/guides/contributing/). A 
+          core team member will be with you shortly to review your PR.
```
Comment (MikeAlhayek) [inline .github/workflows/first_time_contributor.yml:26]:
```suggestion
          Thank you for submitting your first pull request! 🚀 If you haven't already, please take a moment to review our [contribution guide](https://docs.orchardcore.net/en/latest/docs/guides/contributing/). This guide provides helpful information to ensure your contribution aligns with our standards. A core team member will review your pull-request.
```

File: .github/workflows/first_time_contributor.yml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+        FIRST_PR_MERGED_COMMENT: >
+          Congratulations on your first PR merge! 🎉 Thank you for your contribution! We're looking forward to welcome 
+          other contributions of yours in the future. @all-contributors please add @{{ author }} for code.
+        STAR_MESSAGE: If you like Orchard Core, please star our repo!
```
Comment (MikeAlhayek) [inline .github/workflows/first_time_contributor.yml:30]:
```suggestion
        STAR_MESSAGE: If you like Orchard Core, please star our repo and join our [community channels ](https://github.com/OrchardCMS/OrchardCore/?tab=readme-ov-file#get-in-touch)
```

**Review by MikeAlhayek — CHANGES_REQUESTED** [review]

File: .github/workflows/first_time_contributor.yml (hunk 1 above)
Comment (Piedone) [reply .github/workflows/first_time_contributor.yml:22]: Having a very long line makes it a lot harder to work with the text (and auto-wrapping is not really convenient with a whitespace-centric format like YAML). The `>` means that the string will ignore line breaks when processed by the called action, i.e. the actual messages won't have these, it's just for readability in the code file.

Same for https://github.com/OrchardCMS/OrchardCore/pull/15708#discussion_r1563302107.

File: .github/workflows/first_time_contributor.yml (hunk 3 above)
Comment (Piedone) [reply .github/workflows/first_time_contributor.yml:30]: How about linking rather to https://github.com/OrchardCMS/OrchardCore?tab=readme-ov-file#help-and-support, or both?

Comment (MikeAlhayek) [reply .github/workflows/first_time_contributor.yml:30]: I think social media is better so they can stay engaged with us.

File: .github/workflows/first_time_contributor.yml (hunk 1 above)
Comment (MikeAlhayek) [reply .github/workflows/first_time_contributor.yml:22]: You can re-format the text I suggested so it is wrapped as needed

Comment (Piedone) [reply .github/workflows/first_time_contributor.yml]: OK, I added these with slight differences.

**Comment by Piedone** [comment]
Note that https://github.com/OrchardCMS/OrchardCore/pull/15706 needs to be merged first, since this links to a new docs page created there.

**Review by MikeAlhayek — APPROVED** [review]

**Pull request merged**